building/src/sass/building.scss

```diff
@@ -0,0 +1,32 @@
+/**
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+$cu-styles-root: '../../../interface-lib/camelot-unchained/styles/';
+@import "../../node_modules/camelot-unchained/lib/index.scss";
+
+html, body {
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+}
+
+#building-button {
+  position: absolute;
+  left: 0px;
+  bottom: 0px;
+  width: 67px;
+  height: 61px;
+  background: url('../../../interface-lib/camelot-unchained/images/buttons/building-btn.jpg') no-repeat bottom left;
```

Comment:
Link to the icon here: http://camelot-unchained.s3.amazonaws.com/icons/general-ui/building-btn.jpg